python/amico/core.py

```diff
@@ -294,6 +299,9 @@ def fit( self ) :
                         if b0 > 1e-3 :
                             y = y / b0
 
+                    if self.get_config('doMergeB0') and self.scheme.b0_count > 0:
+                        y = np.hstack((np.mean( y[self.scheme.b0_idx] ),y[self.scheme.dwi_idx]))
```

Comment:
If the 'doNormalizeSignal' flag is active, y[0] should be 1.0 and not the value of b0, which is not normalized. And normalizing the b0 to 1.0 if 'doNormalizeSignal' is active introduces problems when 'doSaveCorrectedDWI' is active... I prefered recomputing the mean value of the b0 signal so it doesn't have to check all the combinations of active or inactive flags.